Task_HelloWorld/index.html

```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
```

Comment:
The code is written in HTML but does not seem to be properly formatted. The opening and closing tags should be placed correctly.